addons/editor-dark-mode/addon.json

```diff
@@ -1023,14 +1023,14 @@
         "selector": "#1e1e1e",
         "selector2": "#2e2e2e",
         "selectorSelection": "#111111",
-        "accent": "#111111",
+        "accent": "#1e1e1e",
```

Comment:
Is this for modals or the stage selector? I don't think it should be changed because it also affects sprite info and the costume and sound editors.

Reply:
The stage selector, but I'll change it back.